docs/usage/shoot_workerless.md

```diff
@@ -0,0 +1,29 @@
+# Workerless Shoot
+
+Starting from Gardener version v1.71, users can create a shoot without any workers, known as a "workerless shoot". Previously, users were unable to create shoots without workers, and had to include worker nodes even if they only needed control-plane services. With workerless shoots, Gardener will not create a worker plane.
+
+Here's an example manifest for a workerless shoot:
+
+```yaml
+apiVersion: core.gardener.cloud/v1beta1
+kind: Shoot
+metadata:
+  name: local
+  namespace: garden-local
+spec:
+  seedName: local
+  cloudProfileName: local
+  region: local
+  provider:
+    type: local
+  kubernetes:
+    version: 1.26.0
+```
+
+**_NOTE:_** It's important to note that a workerless shoot cannot be converted to a shoot with workers, and vice versa.
+
+As part of control-plane following components are deployed in the shoot cluster for workerless shoot:
+ - etcds
+ - kube-apiserver
+ - kube-controller-manager
+ - logging and monitoring components
```

Comment:
Some things are missing (GRM, components deployed via `Extension`s that support workerless `Shoot`s), etc.